tests/test-timber.php

```diff
@@ -901,48 +833,6 @@ public function testThePostHook()
         $this->assertSame(3, $the_post_count);
     }
 
-    public function testGetAttachment()
-    {
-        $this->markTestSkipped('@todo seems like a lot of what gets tested here is core WP file mgmt. Is that what we want?');
```

Comment:
> @todo seems like a lot of what gets tested here is core WP file mgmt. Is that what we want?

I agree. Doesn’t seem to test anything worth testing that isn’t covered in other tests.